README.md

```diff
@@ -1,4 +1,4 @@
-# Team Wilder
+# Nextdoor Nature
 
 [Current release](https://team-wilder-proto.pages.dev/)
```

Comment:
Are we going to change this too?

Reply:
Sure why not? Perhaps some wording about the relationship may be in order.

Reply:
I put in a small comment explaining the relationship to WT TeamWilder